pkg/sql/pgwire/testdata/pgtest/portals

```diff
@@ -114,12 +114,14 @@ ReadyForQuery
 {"Type":"ReadyForQuery","TxStatus":"I"}
 
 # Execute a portal with limited rows inside a transaction.
+# This also tests that we can handle Flush during the portal execution.
```

Comment:
I feel like we should probably keep the tests that don't run Flush as well, to make sure we don't regress if someone doesn't provide a Flush.